### ISSUE TITLE
go module broken

### ISSUE THREAD

**Issue by figadore** [pulumi/pulumi-github#276]
### What happened?

Trying to install this go module as instructed in the readme with `$ go get github.com/pulumi/pulumi-github/sdk/v5`, but it fails

### Steps to reproduce

`go get github.com/pulumi/pulumi-github/sdk/v5`

### Expected Behavior

download the module

### Actual Behavior

```
go: downloading github.com/pulumi/pulumi-github/sdk v1.5.0
go: downloading github.com/pulumi/pulumi-github v5.1.0+incompatible
go: module github.com/pulumi/pulumi-github/sdk@upgrade found (v1.5.0), but does not contain package github.com/pulumi/pulumi-github/sdk/v5
```

### Output of `pulumi about`

```
CLI          
Version      3.49.0
Go Version   go1.19.3
Go Compiler  gc

Host     
OS       debian
Version  bookworm/sid
Arch     x86_64

Backend        
Name           pop-os
URL            file://~
User           reese
Organizations  

Pulumi locates its logs in /tmp by default
warning: Failed to read project: no Pulumi.yaml project file found (searching upwards from /some/path/pulumi-gh). If you have not created a project yet, use `pulumi new` to do so: no project file found
warning: Failed to get information about the current stack: no Pulumi.yaml project file found (searching upwards from /some/path/pulumi-gh). If you have not created a project yet, use `pulumi new` to do so: no project file found
```

### Additional context

related: https://github.com/pulumi/templates/issues/481

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by squaremo** [comment]
I can confirm that's what the instructions say at https://github.com/pulumi/pulumi-github#go, and when I try it, I have the same output.

**Comment by guineveresaenger** [comment]
This was fixed as part of #275.
